extensions/amp-story/1.0/test/test-amp-story.js

```diff
@@ -177,6 +201,13 @@ describes.realWin(
 
       pages = await Promise.all(pages);
 
+      // Insert ads
+      const adPage = await createStoryAdPage(adId);
```

Comment:
Would be best to keep this test intact and make a new `describes('ad pages')` with the new tests for the ads logic, so if anything breaks, we know more easily what broke

Reply:
Make sense. Done.